packages/flutter_markdown/lib/src/builder.dart

```diff
@@ -111,6 +109,7 @@ class MarkdownBuilder implements md.NodeVisitor {
     required this.builders,
     required this.paddingBuilders,
     required this.listItemCrossAxisAlignment,
+    required this.customBlockTags,
```

Comment:
Adding a required parameter to a public class is a breaking change, and would need to be versioned as such.

Why does this need to be `required`?

Reply:
Actually, given https://github.com/flutter/packages/pull/5815 why do we need this at all? Isn't this adding a new/different and potentially conflicting way to specific the same information that PR added a way to specify?

Reply:
Actually, I think you're right here.  I didn't realize that API would automatically add to the list. Let me ask the customer why they can't use the existing mechanism.